src/main/java/graphql/schema/GraphqlDirectivesContainerTypeBuilder.java

```diff
@@ -67,7 +78,6 @@ public B replaceDirectives(List<GraphQLDirective> directives) {
     @Deprecated(since = "2022-02-24")
     public B withDirectives(GraphQLDirective... directives) {
         assertNotNull(directives, () -> "directives can't be null");
-        this.directives.clear();
```

Comment:
This method is deprecated, but let's also remove the unintended clear line from here